src/seedu/addressbook/AddressBook.java

```diff
@@ -689,14 +689,18 @@ private static void createFileIfMissing(String filePath) {
 
         showToUser(String.format(MESSAGE_ERROR_MISSING_STORAGE_FILE, filePath));
 
-        try {
+        createNewFile(filePath, storageFile);
+    }
+
+	private static void createNewFile(String filePath, final File storageFile) {
```

Comment:
Indentation here is off, it should be 4 spaces consistently. You can set up your IDE to take care of that.